container-chains/templates/frontier/runtime/src/lib.rs

```diff
@@ -604,6 +605,7 @@ impl<F: FindAuthor<u32>> FindAuthor<H160> for FindAuthorTruncated<F> {
 
 parameter_types! {
     pub BlockGasLimit: U256 = U256::from(BLOCK_GAS_LIMIT);
+    pub const GasLimitPovSizeRatio: u64 = BLOCK_GAS_LIMIT.saturating_div(MAX_POV_SIZE);
```

Comment:
I always forget how does `saturating_div` differ from normal div, so in case anyone is curious:

For unsigned types, it's exactly the same as normal div. It still panics on division by 0.

For signed types, it's the same except one edge case: `i8::MIN / -1` returns `i8::MAX` instead of overflowing (because -256 is a valid value but +256 is not, so it returns +255 instead).

In this case it's a `u64` so it's the same as using `/`, but you can keep it for symmetry with the other operations.